CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+* **Feature Proposals**: These outline new features and capabilities for the IRC protocol.
+
+## Bug Fixes and Changes
+
+A bug fix may be relatively minor, such as [#186](https://github.com/ircv3/ircv3-specifications/pull/186), or may fix issues found with the specification after it has been merged or ratified, such as [#204](https://github.com/ircv3/ircv3-specifications/pull/204).
+
+Minor bug fixes, such as fixing spelling mistakes or making formatting more consistent, may be submitted without much process. However, try to minimise the number of fixes you submit, as having 13 separate PRs to fix a few spelling and formatting issues can become grating to other participants and observers of the IRCv3 WG.
+
+Submitting bug fixes and changes of this type does not have any stakeholder requirements.
+
+Fixes and changes which address the actual behaviour or meaning of a specification *after they have been released*, such as [#204](https://github.com/ircv3/ircv3-specifications/pull/204) generally require 'errata updates'. Examples of errata updates can be found in various specifications, but they basically leave a log as to how a specification has been changed after it was released. Before submitting a fix of this type, you should talk with other members on the [#ircv3](http://ircv3.net/contact.html) channel and ensure there is support for your change.
+
+## Ideas
+
+Feature proposals contributed to the IRCv3-Specifications repository require a concrete specification and at least one reference or existing implementation.
+
+If you have an idea or a protocol proposal doesn't have these things or is not yet at this stage, you should make an issue on the [IRCv3/ideas](https://github.com/ircv3/ideas) repository instead. This repository is intended for working group members, as well as developers and users to be able to suggest ideas that are not yet full realised or without a reference implementation.
+
+Contributing to the [IRCv3/ideas](https://github.com/ircv3/ideas) repository does not have any stakeholder requirements. It was created so that contributors may decide to only review ideas that have already been specified and have support from other stakeholders of the IRCv3 WG, or review those that aren't yet at that stage.
+
+## Feature Proposals
+
+A new feature proposal includes things like [`account-tag`](http://ircv3.net/specs/extensions/account-tag-3.2.html), [`cap-notify`](http://ircv3.net/specs/extensions/cap-notify-3.2.html) and [`echo-message`](http://ircv3.net/specs/extensions/echo-message-3.2.html).
+
+Contributing a proposal to the IRCv3-Specifications repo requires a few things. These include:
+
+### Stakeholder Support
+
+Specifications submitted to this repository should be proposed or sponsored by a stakeholder of the IRCv3 Working Group.
+
+### Concrete Specification
+
+Your specification *must* describe all behaviour that comes with your proposal. You cannot simply say *"Yeah, and this part we'll work out later, it should be fine."*.
```

Comment:
I'm willing to accept this, but I'd rather prefer this to be judged by stakeholders individually for each specification. If the specification works well as-is but could be improved somehow (STS falls into this category) then that should not block pulling the specification. What do you think?